README.md

```diff
@@ -29,7 +29,7 @@ Google APIs Client Libraries, in [Client Libraries Explained][explained].
 * [Quickstart](#quickstart)
   * [Before you begin](#before-you-begin)
   * [Installing the client library](#installing-the-client-library)
-  * [Using the client library](#using-the-client-library)
```

Comment:
This reads like the quickstart got deleted .... (will keep reading)

Reply:
The quickstart isn't deleted, it's just not very robust.

Reply:
Right - but we'd still like to make sure we have one.  Goal is to have a copy/paste snippet on the README that at least helps get folks started :)